eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -99,7 +151,7 @@ test-suite cardano-ledger-alonzo-test
         cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib},
         cardano-ledger-alonzo-test,
         cardano-ledger-binary:{cardano-ledger-binary, testlib},
-        cardano-ledger-core,
+        cardano-ledger-core >=1.0,
```

No need to specify bounds, the library section already sets the bounds.
```suggestion
        cardano-ledger-core,
```